<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerRegistrationFiducialsPanelWidget</class>
 <widget class="QWidget" name="qSlicerRegistrationFiducialsPanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>484</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="PointsTabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="ImagePointsTab">
      <attribute name="title">
       <string>Image Points</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Fiducials:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="qMRMLNodeComboBox" name="ImagePointsAnnotationNodeSelector">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="nodeTypes">
            <stringlist>
             <string>vtkMRMLAnnotationHierarchyNode</string>
            </stringlist>
           </property>
           <property name="addEnabled">
            <bool>true</bool>
           </property>
           <property name="removeEnabled">
            <bool>true</bool>
           </property>
           <property name="renameEnabled">
            <bool>true</bool>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToMinimumContentsLength</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="ImagePointsTable"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="AddRemoveLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="qSlicerMouseModeToolBar" name="MouseModeToolBar">
           <property name="floatable">
            <bool>false</bool>
           </property>
           <addaction name="separator"/>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ClearImagePointsButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PhysicalPointsTab">
      <attribute name="title">
       <string>Physical Points</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="SourceLabel">
           <property name="text">
            <string>Fiducials:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="qMRMLNodeComboBox" name="PhysicalPointsAnnotationNodeSelector">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="nodeTypes">
            <stringlist>
             <string>vtkMRMLAnnotationHierarchyNode</string>
            </stringlist>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToMinimumContentsLength</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="PhysicalPointsTable"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="PhysicalPointsControlLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QPushButton" name="AddPhysicalPointButton">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ClearPhysicalPointsButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Import Option</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <property name="horizontalSpacing">
           <number>-1</number>
          </property>
          <property name="verticalSpacing">
           <number>1</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item row="3" column="0">
           <widget class="QLabel" name="CurrentPositionLabel">
            <property name="text">
             <string>Pointer</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="PosPhysicalLayout">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="qMRMLNodeComboBox" name="TrackerTransformNodeSelector">
              <property name="nodeTypes">
               <stringlist>
                <string>vtkMRMLLinearTransformNode</string>
               </stringlist>
              </property>
              <property name="noneEnabled">
               <bool>true</bool>
              </property>
              <property name="editEnabled">
               <bool>true</bool>
              </property>
              <property name="renameEnabled">
               <bool>true</bool>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToMinimumContentsLength</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="LoadFileLabel">
            <property name="text">
             <string>Current</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="LoadPhysicalPointsButtonLayout">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="PositionXLabel">
              <property name="text">
               <string> X:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="PositionXEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="PositionYLabel">
              <property name="text">
               <string>  Y:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="PositionYEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="PositionZLabel">
              <property name="text">
               <string>  Z:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="PositionZEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="RegistrationSettingLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="NumberOfPointsLabel">
       <property name="text">
        <string>Number of Points:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="NumberOfPoints">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="OutputTransformLabel">
       <property name="text">
        <string>Output Transform:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="qMRMLNodeComboBox" name="OutputTransformNodeSelector">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLLinearTransformNode</string>
        </stringlist>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="registerButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerMouseModeToolBar</class>
   <extends>QToolBar</extends>
   <header>qSlicerMouseModeToolBar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
